### ISSUE TITLE
panic when print SliceNode

### ISSUE THREAD

**Issue by hesining** [antonmedv/expr#411]
```go
func TestRange(t *testing.T) {
	str1 := "[0,1,2,3,4][1:2]" // no panic 
	str1 = "[0,1,2,3,4][1:]"  // panic 
	str1 = "[0,1,2,3,4][:2]"  // panic

	vm, err := expr.Compile(str1)
	assert.NoError(t, err, err)
	fmt.Println(vm.Node.String()) 

	rst, err := expr.Run(vm, nil)
	assert.NoError(t, err, err)
	fmt.Println(rst)
}
```

I check the code, it is caused by the SliceNode String
```go
func (n *SliceNode) String() string {
	return fmt.Sprintf("%s[%s:%s]", n.Node.String(), n.From.String(), n.To.String())  // From or To may be nil
}
```

but I don't know whether to initialize or judge nil value, so I don't try to fix it.

**Comment by antonmedv** [comment]
Nil means no node.

**Comment by hesining** [comment]
> Nil means no node.

sorry , I copy the wrong source code , the raw code is 
```
func (n *SliceNode) String() string {
	return fmt.Sprintf("%s[%s:%s]", n.Node.String(), n.From.String(), n.To.String())  // From or To may be nil, panic here
}
```

**Comment by antonmedv** [comment]
Fixed in cc5d2941ae614305b7a32584730c48d6aef4ced2
